docs/tools/CRAWL.md

```
@@ -0,0 +1,293 @@
+
+### Check Status
+
+Provide `jobId` parameter to check progress and retrieve results.
+
+### Cancel Crawl
+
+Provide `jobId` + `cancel: true` to cancel an in-progress job.
+
+## Workflow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)